test/support/session_case.ex

```diff
@@ -8,7 +8,19 @@ defmodule Wallaby.SessionCase do
   end
 
   setup do
-    {:ok, session} = Wallaby.start_session
+    session_opts = case System.get_env("WEBDRIVER") do
```

Comment:
I realize that this is just for testing so it doesn't matter right now, but if we decide to stick with an implementation like this we should probably prefix the env variable with `WALLABY_`. Totally not important right now though.